test/google_navigation_flutter_test.dart

```diff
@@ -1216,6 +1216,55 @@ void main() {
           // Verify message
           expect(viewId, 0);
         });
+
+        test('set padding', () async {
+          // Create padding
+          EdgeInsets insets = const EdgeInsets.all(10.0);
+
+          // Mock api response
+          when(viewMockApi.setPadding(any, any))
+              .thenAnswer((Invocation _) async => ());
+
+          // Set padding
+          await GoogleMapsNavigationPlatform.instance
+              .setPadding(viewId: 0, padding: insets);
+
+          // Verify correct message sent from view api
+          final VerificationResult result =
+              verify(viewMockApi.setPadding(captureAny, captureAny));
+          final MapPaddingDto paddingMessage =
+              result.captured[1] as MapPaddingDto;
+
+          // Verify message
+          expect(insets.left, paddingMessage.left);
+          expect(insets.right, paddingMessage.right);
+          expect(insets.top, paddingMessage.top);
+          expect(insets.bottom, paddingMessage.bottom);
```

Comment:
As all the edges use same value, this test won't detect if the padding is mapped wrong at conversion
Instead of
`EdgeInsets insets = const EdgeInsets.all(10.0);`
use for example:
`EdgeInsets insets = const EdgeInsets.only(left: 5.0, top: 10.0, right: 15.0, bottom: 20.0);`